utils/dom-purify/index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = process.BUILD_TARGET === 'web' ?
```

Comment:
no. i added that to [poe-ui-builder](https://github.com/poegroup/poe-ui-builder/blob/a1f40c13f4fa3cca68f65a078f9b4e3868f4125d/index.js#L85). it's not really an environment variable